continuous_integration/test_script_windows.bat

```diff
@@ -0,0 +1,39 @@
+@echo off
+
+setlocal enabledelayedexpansion
+
+call conda info
+if %errorlevel% neq 0 exit /b %errorlevel%
```

Comment:
is this necessary to have this at each line with `@echo off`?

Reply:
Apparently, it is mandatory to add it at each line to achieve the same result as `set -e` does in `.sh` scripts. [source](https://stackoverflow.com/questions/734598/how-do-i-make-a-batch-file-terminate-upon-encountering-an-error)